gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v3_16_0/schema.yml

```diff
@@ -0,0 +1,13 @@
+databaseChangeLog:
+  - changeSet:
+      id: 3.16.0
+      author: GraviteeSource Team
+      changes:
+        - addColumn:
+            tableName: ${gravitee_prefix}applications
+            columns:
+              - column:
+                  name: api_key_mode
+                  type: nvarchar(256)
```

Comment:
I don't know if it really matters..
Is there a benefit of reducing the column length, just to store an enum ? Like `nvarchar(16)` ?

Maybe it would save some storage space on some DBMS ?